visualization/app/codeCharta/util/customConfigHelper.spec.ts

```diff
@@ -500,11 +503,42 @@ describe("CustomConfigHelper", () => {
 	})
 
 	describe("importCustomConfigs", () => {
+		it("should import Configs in SINGLE Mode and change mode to DELTA", () => {
```

Comment:
Did you mean `should import Configs in SINGLE Mode and change mode to **MULTIPLE**`?

Reply:
You're right!